documentation/modules/ROOT/pages/transformations/outbox-event-router.adoc

```diff
@@ -245,9 +304,22 @@ value.converter.delegate.converter.type.schemas.enable=false
 The delegate `Converter` implementation is specified by the `delegate.converter.type` option.
 If any extra configuration options are needed by the converter, they can also be specified, such as the disablement of schemas shown above using `schemas.enable=false`.
 
+Furthermore, the following example illustrates how to use a delegate converter in Avro format with Schema Registry:
```

Comment:
```suggestion
The following example illustrates how to use a delegate converter in Avro format with Schema Registry:
```

Reply:
Should the reference to _Schema Registry_ be changed to _Apicurio Registry_?

Reply:
I don't have access to a running instance of Apicurio Registry in order to test the setup and provide an example. But yes, if you think it would be better to document Apicurio, we can do it. Would the example below would still be valid?

Reply:
Hi, yes, I'd prefer to have either Apicurio registry example or both. Please check https://github.com/debezium/debezium/blob/main/debezium-connector-postgres/pom.xml#L227 how do we start the simplest Apicurio registry deployment. It can be started easily via `docker run`

Reply:
Added example for apicurio